.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Reproducibility**
+- [ ] I can reproduce the bug consistently.
```

Comment:
Perhaps:
- Can be reproduced consistently
- Difficult to reproduce
- Unable to reproduce

?

Reply:
good suggestions. added in latest commit.